proto/autokitteh/sessions/v1/session.proto

```diff
@@ -36,7 +36,7 @@ message SessionState {
       (buf.validate.field).map.keys.string.min_len = 1,
       (buf.validate.field).map.values.required = true
     ];
-    values.v1.Value return_value = 3 [(buf.validate.field).required = true];
+    values.v1.Value return_value = 3;
```

Comment:
return value not required?

Reply:
should session log return value?

Reply:
I'll let @itayd decide on that and follow-up in a separate PR (keep? deprecate field? delete field?).

Reply:
this is odd - this should always be specified. @daabr can you reporoduce?

Reply:
@itayd yes - if you add this check back, the system test `connections/var.txtar` and the sessions test `activity_global.txtar` both fail with the same problem related to getting session logs.

See this commit's GitHub action failures as an example: https://github.com/autokitteh/autokitteh/actions/runs/9539935684/job/26290934987

Reply:
please create a ticket as a bug on me for this

Reply:
https://linear.app/autokitteh/issue/ENG-1040/session-state-return-value-proto-validation-check